### ISSUE TITLE
Paginator: Accessible buttons

### ISSUE THREAD

**Issue by melloware** [primefaces/primereact#2827]
### Describe the bug

All paginator icon buttons do not have `aria-label` to identify them to screenreaders.

Similar to: https://github.com/primefaces/primereact/issues/1432

The same fix should be applied for Paginator.

### Reproducer

https://primefaces.org/primereact/datatable/paginator/

### PrimeReact version

8.0.0

### React version

18.x

### Language

ALL

### Build / Runtime

Next.js

### Browser(s)

ALL

### Steps to reproduce the behavior

View the button in the DOM inspector or WAVE Web Accessibility Evaluation Tool

### Expected behavior

When navigating to a button, descriptive text must be presented to screen reader users to indicate the function of the button.

**Use WAVE Web Accessibility Evaluation Tool**
